axelrod/tests/strategies/test_stalker.py

```diff
@@ -21,26 +21,39 @@ class TestStalker(TestPlayer):
     }
 
     def test_strategy(self):
+        
         # Start with cooperation
         self.first_play_test(C)
 
         # current_average_score > very_good_score
-        self.responses_test([D], [C] * 2 + [D] * 4, [D] * 2 + [C] * 4)
```

Comment:
@marcharper I try changing it to its original state and its still not testing the current_average > very_good_score. Any guidance on how to fix it. Thanks